composer.json

```diff
@@ -34,7 +34,7 @@
     },
     "require-dev": {
         "phpunit/phpunit": "~8.0",
-        "vimeo/psalm": "~3.7.0"
+        "vimeo/psalm": "^3.7"
```

Comment:
To install latest `Psalm` version, it should use `^3.7` to define the `psalm` dependency.